packages/sdks/output/angular/ng-package.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "./node_modules/ng-packagr/ng-package.schema.json",
-  "dest": "lib/browser",
+  "dest": "lib/edge",
```

Comment:
can you explain this change?

Reply:
i think after Anagha's edge support got it in, when we run yarn g:nx build inside the angular output folder - it runs all of the builds in sequence and this gets updated one by one but technically this shouldn't affect anything